katib/base/katib-ui-virtual-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: katib-ui
+  namespace: kubeflow
```

Comment:
we'd better remove `namespace` for [Best-practice](https://github.com/kubeflow/manifests#Best-practice-for-kustomize-targets)

Reply:
removed.